### PR TITLE
0.8.3 release blog post

### PR DESCRIPTION
**Pull request by aravindavk**
Signed-off-by: Aravinda Vishwanathapura <aravinda@kadalu.io>

### CODE REVIEW

File: content/blog/2021-06-28-kadalu-storage-0.8.3.adoc [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    --device kube-nodename1:/dev/vdc                       \
+    --device kube-nodename2:/dev/vdc                       \
+    --device kube-nodename3:/dev/vdc
+----
+
+=== Distributed Volume support for External Volumes
+
+Early releases of Kadalu used xfs quota in the Storage backend for each PVs created. With that supporting Distribute Storage was a challenge. With the version 0.8.0, the Simple Quota feature was introduced for the Storage volumes managed by Kadalu Operator. But the Simple Quota feature is not available with GlusterFS releases yet! So Distribute was not supported for the external GlusterFS Volumes.
+
+With this release, Distribute Volume support is added by using the GlusterFS Quota feature itself. Thanks to **Yugo Yamauchi from Hitachi** for this wonderful enhancement. Read more details about this enhancement https://kadalu.io/rfcs/0007-Using-GlusterFS-directory-quota-for-external-gluster-volumes.html[here].
```
Comment (amarts) [inline content/blog/2021-06-28-kadalu-storage-0.8.3.adoc:28]: Do we need 'link:http....' here or just https:// works?

File: content/blog/2021-06-28-kadalu-storage-0.8.3.adoc [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+additional information.
+
+== Contributors to this release (8)
+
+A huge thanks go to all the awesome people who made this release
+possible.
+
+* https://github.com/amarts[Amar Tumballi]
+* https://github.com/aravindavk[Aravinda Vishwanathapura]
+* https://github.com/hunter86bg[hunter86bg]
```
Comment (amarts) [inline content/blog/2021-06-28-kadalu-storage-0.8.3.adoc]: name is: Strahil Nikolov

**Review by amarts — APPROVED** [review]

File: content/blog/2021-06-28-kadalu-storage-0.8.3.adoc (hunk 1 above)
Comment (aravindavk) [reply content/blog/2021-06-28-kadalu-storage-0.8.3.adoc:28]: It works. Only for the relative links that doesn't have `http(s)` prefix `link:` is required.

File: content/blog/2021-06-28-kadalu-storage-0.8.3.adoc (hunk 2 above)
Comment (aravindavk) [reply content/blog/2021-06-28-kadalu-storage-0.8.3.adoc]: Updated.

**Pull request merged**